src/web/app/pages-student/student-course-details-page/__snapshots__/student-course-details-page.component.spec.ts.snap

```diff
@@ -128,71 +133,6 @@ exports[`StudentCourseDetailsPageComponent should snap with populated fields 1`]
     </div>
   </div><div
     class="section"
-  >
```

Comment:
I think you should modify the snapshot test accordingly?

Reply:
Done!